actions/stop-all.go

```diff
@@ -20,24 +20,21 @@ import (
 
 //StopAllCommand to stop codewind and project containers
 func StopAllCommand() {
-	containerArr := [2]string{}
+	containerArr := [4]string{}
 	containerArr[0] = "codewind-pfe"
 	containerArr[1] = "codewind-performance"
-	containerArr[0] = "cw-"
-	containerArr[1] = "appsody"
+	containerArr[2] = "cw-"
+	containerArr[3] = "appsody"
 
 	containers := utils.GetContainerList()
 
 	fmt.Println("Stopping Codewind and Project containers")
 	for _, container := range containers {
 		for _, key := range containerArr {
 			if strings.HasPrefix(container.Image, key) {
-				fmt.Println(container)
-				// stop appsody containers with "cw-" in their name, and all other cw containers
-				if key != "appsody" || strings.Contains(container.Names[0], "cw-") {
-					fmt.Println("Stopping container ", container.Names, "... ")
-					utils.StopContainer(container)
-				}
+				fmt.Println("Stopping container ", container.Names[0], "... ")
```

Comment:
The check for the `cw-` container prefix (for appsody) was put in to avoid stopping appsody containers that are started outside of Codewind

Reply:
ok, will add back in